src/Interceptor/JsonSchemaInterceptor.php

```diff
@@ -48,26 +48,29 @@ public function __construct($schemaDir, $validateDir)
      */
     public function invoke(MethodInvocation $invocation)
     {
+        /* @var $jsonSchema JsonSchema */
+        $jsonSchema = $invocation->getMethod()->getAnnotation(JsonSchema::class);
+
+        if ($jsonSchema->params) {
+            $arguments = $this->getNamedArguments($invocation);
+            $this->validateRequest($jsonSchema, $arguments);
+        }
+
+        /** @var $ro ResourceObject */
```

Comment:
Not `/** @var`, `/* @var` should  be prefered.

Passed `composer cs` ?

Reply:
@koriym Yes, all the checks passed, but I think all the coding styles should be identical so I corrected it.